<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>588</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>餐馆信息管理系统</string>
  </property>
  <property name="windowIcon">
   <iconset resource="res.qrc">
    <normaloff>:/Logo.jpg</normaloff>:/Logo.jpg</iconset>
  </property>
  <property name="toolTip">
   <string>这是餐厅信息管理系统</string>
  </property>
  <property name="styleSheet">
   <string notr="true">/* Main window background color with gradient */
/* Common background widget style */

/* Common background widget style */
QWidget#centralwidget {
    background: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(240, 245, 235, 1), stop:0.5 rgba(255, 255, 255, 1), stop:1 rgba(240, 245, 235, 1)); /* Gradient from light green to white to light green */
    color: #333333; /* Dark Grey */
    font-family: &quot;Segoe UI&quot;, sans-serif;
    font-size: 10pt;
}


/* General widget style */
QWidget {
    background-color: #FFFFFF; /* White */
    color: #333333; /* Dark Grey */
    font-family: &quot;Segoe UI&quot;, sans-serif;
    font-size: 10pt;
}

/* Tree widget style */
QTreeWidget {
    background: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 #FFFFFF, stop:1 #F0F0F0); /* White to Very Light Grey Gradient */
    color: #333333; /* Dark Grey */
    border: 1px solid #CCCCCC; /* Light Grey Border */
}

/* Buttons common style */
QPushButton {
    background: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 #E0E0E0, stop:1 #FFFFFF); /* Light Grey to White Gradient */
    color: #333333; /* Dark Grey */
    border: 1px solid #CCCCCC; /* Light Grey Border */
    border-radius: 4px;
    padding: 5px;
    font-family: &quot;Segoe UI&quot;, sans-serif;
    font-size: 10pt;
}

QPushButton:hover {
    background-color: #D6D6D6; /* Slightly Darker Grey */
}

QPushButton:pressed {
    background-color: #BFBFBF; /* Darker Grey */
}

/* Specific buttons adjustments */
QPushButton#btn_exit {
    background-color: #FF6B6B; /* Light Red */
    border: 1px solid #FF9999; /* Light Red Border */
}

QPushButton#btn_exit:hover {
    background-color: #FF5252; /* Darker Red */
}

QPushButton#btn_exit:pressed {
    background-color: #FF3B3B; /* Even Darker Red */
}

QPushButton#btn_queue, QPushButton#btn_cashier {
    background: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 #E0E0E0, stop:1 #FFFFFF); /* Light Grey to White Gradient */
    color: #333333; /* Dark Grey */
    border: 1px solid #CCCCCC; /* Light Grey Border */
    border-radius: 4px;
    padding: 5px;
    font-family: &quot;Segoe UI&quot;, sans-serif;
    font-size: 10pt;
}

QPushButton#btn_queue:hover, QPushButton#btn_cashier:hover {
    background-color: #D6D6D6; /* Slightly Darker Grey */
}

QPushButton#btn_queue:pressed, QPushButton#btn_cashier:pressed {
    background-color: #BFBFBF; /* Darker Grey */
}

/* Labels style */
QLabel {
    color: #333333; /* Dark Grey */
    font-family: &quot;Segoe UI&quot;, sans-serif;
    font-size: 10pt;
}

QLabel#label {
    font-size: 12pt;
    font-family: &quot;Segoe UI Semibold&quot;, sans-serif;
}

/* Line edits style */
QLineEdit {
    background-color: #FFFFFF; /* White */
    color: #333333; /* Dark Grey */
    border: 1px solid #CCCCCC; /* Light Grey Border */
    border-radius: 4px;
    padding: 4px;
    font-family: &quot;Segoe UI&quot;, sans-serif;
    font-size: 10pt;
}

/* Table widgets style */
QTableWidget {
    background-color: #FFFFFF; /* White */
    color: #333333; /* Dark Grey */
    gridline-color: #CCCCCC; /* Light Grey */
    border: 1px solid #CCCCCC; /* Light Grey Border */
    font-family: &quot;Segoe UI&quot;, sans-serif;
    font-size: 10pt;
}

QHeaderView::section {
    background: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 #E0E0E0, stop:1 #FFFFFF); /* Light Grey to White Gradient */
    color: #333333; /* Dark Grey */
    padding: 4px;
    border: 1px solid #CCCCCC; /* Light Grey Border */
    font-family: &quot;Segoe UI Semibold&quot;, sans-serif;
}

/* Menu bar and status bar style */
QMenuBar {
    background: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 #F7F7F7, stop:1 #FFFFFF); /* Light Grey to White Gradient */
    color: #333333; /* Dark Grey */
    font-family: &quot;Segoe UI&quot;, sans-serif;
    font-size: 10pt;
}

QStatusBar {
    background: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 #F7F7F7, stop:1 #FFFFFF); /* Light Grey to White Gradient */
    color: #333333; /* Dark Grey */
    font-family: &quot;Segoe UI&quot;, sans-serif;
    font-size: 10pt;
}
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>60</y>
      <width>651</width>
      <height>641</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <widget class="QWidget" name="page1">
     <widget class="QPushButton" name="btn_addDish">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>0</y>
        <width>61</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>增加</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_updDish">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>0</y>
        <width>61</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>修改</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_delDish">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>0</y>
        <width>61</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>删除</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_search">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>0</y>
        <width>113</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_search">
      <property name="geometry">
       <rect>
        <x>570</x>
        <y>0</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>搜索</string>
      </property>
     </widget>
     <widget class="QTableWidget" name="tableWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>631</width>
        <height>451</height>
       </rect>
      </property>
      <attribute name="horizontalHeaderStretchLastSection">
       <bool>true</bool>
      </attribute>
      <column>
       <property name="text">
        <string>菜品编号</string>
       </property>
       <property name="font">
        <font>
         <underline>false</underline>
        </font>
       </property>
      </column>
      <column>
       <property name="text">
        <string>名称</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>分类</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>主要材料</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>价格</string>
       </property>
      </column>
     </widget>
     <widget class="QPushButton" name="btn_menu_refresh">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>0</y>
        <width>61</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>刷新</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2">
     <widget class="QTableWidget" name="tableWidget_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>30</y>
        <width>641</width>
        <height>441</height>
       </rect>
      </property>
      <attribute name="horizontalHeaderStretchLastSection">
       <bool>true</bool>
      </attribute>
     </widget>
     <widget class="QPushButton" name="btn_addTable">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>0</y>
        <width>61</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>增加</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_updTable">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>0</y>
        <width>61</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>修改</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_delTable">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>0</y>
        <width>61</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>删除</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_search2">
      <property name="geometry">
       <rect>
        <x>570</x>
        <y>0</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>搜索</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_search2">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>0</y>
        <width>121</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_table_refresh">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>0</y>
        <width>61</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>刷新</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_3">
     <widget class="QPushButton" name="btn_serve_dish">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>0</y>
        <width>92</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>上菜</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_refresh_order">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>0</y>
        <width>92</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>刷新</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_cancel_order">
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>0</y>
        <width>92</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>取消点菜</string>
      </property>
     </widget>
     <widget class="QTableWidget" name="tableWidget_3">
      <property name="geometry">
       <rect>
        <x>15</x>
        <y>31</y>
        <width>631</width>
        <height>431</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_previous_order">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>0</y>
        <width>92</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>上菜记录</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_4">
     <widget class="QPushButton" name="btn_checkout">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>0</y>
        <width>92</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>结账</string>
      </property>
     </widget>
     <widget class="QTableWidget" name="tableWidget_4">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>621</width>
        <height>431</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_refresh_cashier">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>0</y>
        <width>92</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>刷新</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_empty"/>
    <widget class="QWidget" name="page5">
     <widget class="QTableWidget" name="tableWidget_ordermenu">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>351</width>
        <height>401</height>
       </rect>
      </property>
     </widget>
     <widget class="QTableWidget" name="tableWidget_order_table">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>50</y>
        <width>281</width>
        <height>401</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_new_order">
      <property name="geometry">
       <rect>
        <x>530</x>
        <y>10</y>
        <width>92</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>点菜</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_order_menu">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>10</y>
        <width>92</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>刷新</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>801</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">/* Common background widget style */
QWidget {
    background: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(240, 245, 235, 1), stop:0.5 rgba(255, 255, 255, 1), stop:1 rgba(240, 245, 235, 1)); /* Gradient from light green to white to light green */
    color: #333333; /* Dark Grey */
    font-family: &quot;Segoe UI&quot;, sans-serif;
    font-size: 10pt;
}
</string>
    </property>
    <widget class="QPushButton" name="btn_exit">
     <property name="geometry">
      <rect>
       <x>710</x>
       <y>10</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">/* Common button style */
QPushButton {
    background: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(85, 170, 127, 1), stop:1 rgba(100, 190, 145, 1)); /* Gradient from primary to slightly lighter green */
    color: #FFFFFF; /* White text for contrast */
    border: 1px solid #5DAE89; /* Darker green border */
    border-radius: 8px; /* Slightly more rounded corners */
    padding: 6px 12px; /* Increased padding for a more comfortable touch target */
    font-family: &quot;Segoe UI&quot;, sans-serif;
    font-size: 10pt;
    font-weight: bold; /* Bold text for emphasis */
}

QPushButton:hover {
    background: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(100, 190, 145, 1), stop:1 rgba(85, 170, 127, 1)); /* Reverse gradient for hover effect */
}

QPushButton:pressed {
    background: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(70, 150, 110, 1), stop:1 rgba(85, 170, 127, 1)); /* Darker gradient for pressed effect */
}
</string>
     </property>
     <property name="text">
      <string>退出</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>650</x>
       <y>20</y>
       <width>61</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>admin</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>610</x>
       <y>20</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/user.jpg);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>0</y>
       <width>221</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>等线 Light</family>
       <pointsize>14</pointsize>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 14pt &quot;等线 Light&quot;;
background-color: transparent;</string>
     </property>
     <property name="text">
      <string>餐馆信息管理系统</string>
     </property>
    </widget>
   </widget>
   <widget class="QTreeWidget" name="treeWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>50</y>
      <width>131</width>
      <height>491</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <attribute name="headerVisible">
     <bool>false</bool>
    </attribute>
    <column>
     <property name="text">
      <string notr="true">1</string>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="btn_1">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>50</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">/* Common button style */
QPushButton {
    background: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(85, 170, 127, 1), stop:1 rgba(100, 190, 145, 1)); /* Gradient from primary to slightly lighter green */
    color: #FFFFFF; /* White text for contrast */
    border: 1px solid #5DAE89; /* Darker green border */
    border-radius: 8px; /* Slightly more rounded corners */
    padding: 6px 12px; /* Increased padding for a more comfortable touch target */
    font-family: &quot;Segoe UI&quot;, sans-serif;
    font-size: 10pt;
    font-weight: bold; /* Bold text for emphasis */
}

QPushButton:hover {
    background: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(100, 190, 145, 1), stop:1 rgba(85, 170, 127, 1)); /* Reverse gradient for hover effect */
}

QPushButton:pressed {
    background: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(70, 150, 110, 1), stop:1 rgba(85, 170, 127, 1)); /* Darker gradient for pressed effect */
}
</string>
    </property>
    <property name="text">
     <string>菜品管理</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>80</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">/* Common button style */
QPushButton {
    background: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(85, 170, 127, 1), stop:1 rgba(100, 190, 145, 1)); /* Gradient from primary to slightly lighter green */
    color: #FFFFFF; /* White text for contrast */
    border: 1px solid #5DAE89; /* Darker green border */
    border-radius: 8px; /* Slightly more rounded corners */
    padding: 6px 12px; /* Increased padding for a more comfortable touch target */
    font-family: &quot;Segoe UI&quot;, sans-serif;
    font-size: 10pt;
    font-weight: bold; /* Bold text for emphasis */
}

QPushButton:hover {
    background: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(100, 190, 145, 1), stop:1 rgba(85, 170, 127, 1)); /* Reverse gradient for hover effect */
}

QPushButton:pressed {
    background: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(70, 150, 110, 1), stop:1 rgba(85, 170, 127, 1)); /* Darker gradient for pressed effect */
}
</string>
    </property>
    <property name="text">
     <string>餐桌管理</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_3">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>110</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">/* Common button style */
QPushButton {
    background: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(85, 170, 127, 1), stop:1 rgba(100, 190, 145, 1)); /* Gradient from primary to slightly lighter green */
    color: #FFFFFF; /* White text for contrast */
    border: 1px solid #5DAE89; /* Darker green border */
    border-radius: 8px; /* Slightly more rounded corners */
    padding: 6px 12px; /* Increased padding for a more comfortable touch target */
    font-family: &quot;Segoe UI&quot;, sans-serif;
    font-size: 10pt;
    font-weight: bold; /* Bold text for emphasis */
}

QPushButton:hover {
    background: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(100, 190, 145, 1), stop:1 rgba(85, 170, 127, 1)); /* Reverse gradient for hover effect */
}

QPushButton:pressed {
    background: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(70, 150, 110, 1), stop:1 rgba(85, 170, 127, 1)); /* Darker gradient for pressed effect */
}
</string>
    </property>
    <property name="text">
     <string>上菜</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_5">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>140</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">/* Common button style */
QPushButton {
    background: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(85, 170, 127, 1), stop:1 rgba(100, 190, 145, 1)); /* Gradient from primary to slightly lighter green */
    color: #FFFFFF; /* White text for contrast */
    border: 1px solid #5DAE89; /* Darker green border */
    border-radius: 8px; /* Slightly more rounded corners */
    padding: 6px 12px; /* Increased padding for a more comfortable touch target */
    font-family: &quot;Segoe UI&quot;, sans-serif;
    font-size: 10pt;
    font-weight: bold; /* Bold text for emphasis */
}

QPushButton:hover {
    background: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(100, 190, 145, 1), stop:1 rgba(85, 170, 127, 1)); /* Reverse gradient for hover effect */
}

QPushButton:pressed {
    background: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(70, 150, 110, 1), stop:1 rgba(85, 170, 127, 1)); /* Darker gradient for pressed effect */
}
</string>
    </property>
    <property name="text">
     <string>点菜</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_queue">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>170</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">/* Common button style */
QPushButton {
    background: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(85, 170, 127, 1), stop:1 rgba(100, 190, 145, 1)); /* Gradient from primary to slightly lighter green */
    color: #FFFFFF; /* White text for contrast */
    border: 1px solid #5DAE89; /* Darker green border */
    border-radius: 8px; /* Slightly more rounded corners */
    padding: 6px 12px; /* Increased padding for a more comfortable touch target */
    font-family: &quot;Segoe UI&quot;, sans-serif;
    font-size: 10pt;
    font-weight: bold; /* Bold text for emphasis */
}

QPushButton:hover {
    background: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(100, 190, 145, 1), stop:1 rgba(85, 170, 127, 1)); /* Reverse gradient for hover effect */
}

QPushButton:pressed {
    background: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(70, 150, 110, 1), stop:1 rgba(85, 170, 127, 1)); /* Darker gradient for pressed effect */
}
</string>
    </property>
    <property name="text">
     <string>排队</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_cashier">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>200</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">/* Common button style */
QPushButton {
    background: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(85, 170, 127, 1), stop:1 rgba(100, 190, 145, 1)); /* Gradient from primary to slightly lighter green */
    color: #FFFFFF; /* White text for contrast */
    border: 1px solid #5DAE89; /* Darker green border */
    border-radius: 8px; /* Slightly more rounded corners */
    padding: 6px 12px; /* Increased padding for a more comfortable touch target */
    font-family: &quot;Segoe UI&quot;, sans-serif;
    font-size: 10pt;
    font-weight: bold; /* Bold text for emphasis */
}

QPushButton:hover {
    background: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(100, 190, 145, 1), stop:1 rgba(85, 170, 127, 1)); /* Reverse gradient for hover effect */
}

QPushButton:pressed {
    background: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(70, 150, 110, 1), stop:1 rgba(85, 170, 127, 1)); /* Darker gradient for pressed effect */
}
</string>
    </property>
    <property name="text">
     <string>收银</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
